addons/web/static/src/legacy/scss/utils.scss

```diff
@@ -147,7 +147,7 @@
 
 @mixin o-field-pointer() {
     // Force `pointer`cursor on  inputs and labels
-    &:not(:disabled):not(.disabled) {
+    &:not(:disabled):not(.disabled):not(:has(:disabled)) {
```

Comment:
`:has()` is lovely... but still not supported by Firefox and very new for the others (cf. https://caniuse.com/css-has)

Reply:
@Iucapad NB: I know you've seen it (cf. your comment) but Chrome 105+ and Safari 15.4+ is still very (very) new...

Reply:
Yes, I see that as fast workaround to handle most (but not all) browsers directly in 16.0.
We'll handle checkboxes in a more consistent way in task-3094083

Reply:
@stefanorigano I understand your point, but I think that saying "handle **most** (but not all) browsers" is kind of "not wanting to face the reality" (aka it works in Chrome, so :shrug: )... `:has()` isn't used in the codebase for this reason (even knowing that it could solve/simplify certain rules).

Nevertheless, could we move the `:not(:disabled)` selector to the children instead (i.e. `input`) ?

And again, `:has()` is a really cool feature, and I'm thrilled to use it ! :heart:

Reply:
@pparidans , @Iucapad ,

task-3094083 is actually almost done... now that I see the code I think that it should be possible to merge it in 16.0 directly.

Before proceeding further with this PR, I suggest to have a look at @mano-odoo one:
- https://github.com/odoo/odoo/pull/107520